### PR TITLE
s390x is not be able to run grub_test

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

File: products/opensuse/main.pm [hunk 1]
```diff
@@ -344,7 +344,7 @@ sub load_reboot_tests() {
     if (check_var('ARCH', 's390x')) {
         loadtest "installation/reconnect_s390";
     }
-    if (installyaststep_is_applicable()) {
+    if (installyaststep_is_applicable() && !check_var('ARCH', 's390x')) {
```
Comment (okurz) [inline products/opensuse/main.pm]: why should first_boot or boot_encrypt not apply to s390x? why not put it into installyaststep_is_applicable with a comment why it does not apply for s390x?

Comment (Soulofdestiny) [inline products/opensuse/main.pm]: first one is true, but how to extract it to installyaststep_is_applicable and explicitly excluding this one test but not the others?

**Comment by Soulofdestiny** [comment]
updated

File: products/opensuse/main.pm [hunk 2]
```diff
@@ -345,7 +345,9 @@ sub load_reboot_tests() {
         loadtest "installation/reconnect_s390";
     }
     if (installyaststep_is_applicable()) {
-        loadtest "installation/grub_test";
+        if (!check_var('ARCH', 's390x')) {
```
Comment (okurz) [inline products/opensuse/main.pm]: the "why" is missing

**Comment by Soulofdestiny** [comment]
the "why" added

**Pull request merged**